pennylane/devices/qubit/simulate.py

```diff
@@ -335,6 +368,353 @@ def simulate_partial(k):
     )
 
 
+# pylint: disable=too-many-arguments, dangerous-default-value
```

Comment:
I would be extremely cautious about silencing `dangerous-default-value`.  While a lot of the other pylint comment are purely stylistic, this is one that can lead to very difficult to debug debugs and near impossible to test behavior.

Reply:
Good point. This is an obsolete disabled warning so I'll remove it.